src/integrationTest/resources/db/testmigration/V1_1__init_tables.sql

```diff
@@ -89,7 +89,7 @@ create table case_worker_idam_role_assoc(
 
 create table case_worker_audit (
     job_id serial,
-    authenticated_user_id varchar(32),
+    authenticated_user_id varchar(36),
```

Comment:
I think it should be 64, instead of 36.

Reply:
It is fine it UUID and it should be max 36

Reply:
Thanks.